ui/src/components/store.js

```diff
@@ -19,9 +19,9 @@ import { jobReducer } from './job/jobReducer';
 import jobWatcher from './job/jobMiddleware';
 
 const initialiseSagaMiddleware = createSagaMiddleware();
+const isDev = (process.env.NODE_ENV == 'development');
 
-const middleware = [
-  ...getDefaultMiddleware(),
+const middleware = (isDev) ? [...getDefaultMiddleware(), initialiseSagaMiddleware] : [
```

Comment:
What is this change for?

Reply:
getDefaultMiddleware check lots of dev stuff like is the state immutable.
it slow down considerably the UI. 

Good for dev bad for the rest